src/config/settings/base.py

```diff
@@ -198,6 +198,8 @@
         # https://docs.djangoproject.com/en/dev/ref/settings/#dirs
         "DIRS": [
             str(BASE_DIR / "templates"),
+            # Needed for django-machina to find the correct templates without cluttering the templates base directory
+            str(BASE_DIR / "templates/forum"),
```

Comment:
This looks like it's necessary and also good way to separate forum specific html files for the future too without clustering!